sqlparse/sql.py

```diff
@@ -433,6 +433,10 @@ def get_type(self):
         elif token.ttype in (T.Keyword.DML, T.Keyword.DDL):
             return token.normalized
 
+        # duckdb supports FROM-first syntax, PIVOT statements
+        elif token.normalized in {'FROM', 'PIVOT', 'PIVOT_WIDER', 'UNPIVOT'}:
+            return 'SELECT'
```

Comment:
It's a little weird to have this in here, but I think it's worth it because this scopes it to specifically looking at the first token in a statement. I don't want to change the interpretation of "from" too much except to say that it's valid for a select query to start with it, and that statements starting with the pivot keywords are forms of select statements.

I was previously doing this mapping to select in sql-metadata (and still am actually because it looks at tokens directly instead of `Statement.get_type()`), but in the AST service we look at each (sqlparse) statement and inspect its type before parsing (each statement separately) with sql-metadata